docs/source/en/tutorials/using_peft_for_inference.md

```diff
@@ -75,6 +75,12 @@ image
 
 ![pixel-art](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/peft_integration/diffusers_peft_lora_inference_12_1.png)
 
+<Tip>
+
+By default, if we detect the most up-to-date versions of `peft` (0.13.1) and `transformers` (4.46.0), we set `low_cpu_mem_usage=True` to speed up the loading time of LoRA checkpoints. 
```

Comment:
In the future, the version numbers will change so I don't think we don't need to include them.

```suggestion
By default, if the most up-to-date versions of PEFT and Transformers are detected, `low_cpu_mem_usage` is set to `True` to speed up the loading time of LoRA checkpoints. 
```